share/shwap/p2p/shrex/metrics.go

```diff
@@ -11,63 +11,92 @@ import (
 	"github.com/celestiaorg/celestia-node/libs/utils"
 )
 
-var meter = otel.Meter("shrex/eds")
+var meter = otel.Meter("shrex")
 
 type status string
 
 const (
-	StatusBadRequest  status = "bad_request"
-	StatusSendRespErr status = "send_resp_err"
-	StatusSendReqErr  status = "send_req_err"
-	StatusReadRespErr status = "read_resp_err"
-	StatusInternalErr status = "internal_err"
-	StatusNotFound    status = "not_found"
-	StatusTimeout     status = "timeout"
-	StatusSuccess     status = "success"
-	StatusRateLimited status = "rate_limited"
+	statusBadRequest  status = "bad_request"
+	statusSendRespErr status = "send_resp_err"
+	statusSendReqErr  status = "send_req_err"
+	statusReadRespErr status = "read_resp_err"
+	statusReadReqErr  status = "read_req_err"
+	statusInternalErr status = "internal_err"
+	statusNotFound    status = "not_found"
+	statusTimeout     status = "timeout"
+	statusSuccess     status = "success"
+	statusRateLimited status = "rate_limited"
```

Comment:
Will be easier if we split into 2 group: client and server statuses

Reply:
Not sure if it's worth it. The statuses are self-described.

Reply:
They are giant piece of statuses, it is hard to read. Server and client statuses are out of order

Reply:
let me sort them